### PR TITLE
Migrate deposit payments to TaskPayment

### PR DESCRIPTION
**Pull request by jiivan**

### CODE REVIEW

File: golem/model.py [hunk 1]
```diff
@@ -359,24 +338,6 @@ def missing_amount(self):
         # pylint: disable=no-member
         return self.expected_amount - self.wallet_operation.amount
 
-
-class DepositPayment(BaseModel):
```
Comment (Krigpl) [inline golem/model.py:363]: Either don't delete the model or delete the corresponding table in the migration as well.
This is the reason https://github.com/golemfactory/golem/issues/4341 happens.

**Review by Krigpl — COMMENTED** [review]
I don't think it's safe to modify existing migrations.

**Comment by jiivan** [comment]
> I don't think it's safe to modify existing migrations.

I also think so, but unfortunatelly they used custom fields.

**Review by Krigpl — APPROVED** [review]
waffle

**Pull request merged**